Recipes.Test/JsonConfiguration/CompactJsonSerializerTest.cs

```diff
@@ -0,0 +1,137 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="CompactJsonSerializerTest.cs">
+//     Copyright (c) 2016. All rights reserved. Licensed under the MIT license. See LICENSE file in
+//     the project root for full license information.
+// </copyright>
+// <auto-generated>
+// Sourced from NuGet package. Will be overwritten with package update except in Spritely.Recipes source.
+// </auto-generated>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace Spritely.Recipes.Test
+{
+    using System.ComponentModel;
+    using System.Diagnostics.CodeAnalysis;
+    using System.Linq;
+    using Newtonsoft.Json.Linq;
+
+    using NUnit.Framework;
+
+    [TestFixture]
+    public class CompactJsonSerializerTest
+    {
+        [Test]
+        public void SerializeObject_without_type_serializes_to_json_using_CompactSerializerSettings()
+        {
+            var dog = new Dog(5, "spud", FurColor.Brindle);
+
+            var json = CompactJsonSerializer.SerializeObject(dog);
+
+            Assert.That(json, Is.EqualTo("{\"name\":\"spud\",\"furColor\":\"brindle\",\"dogTag\":null,\"age\":5}"));
+        }
+
+        [Test]
+        public void SerializeObject_with_type_serializes_to_json_using_CompactSerializerSettings()
+        {
+            var dog = new Dog(5, "spud", FurColor.Brindle);
+
+            var json = CompactJsonSerializer.SerializeObject(dog, typeof(Animal));
+
+            Assert.That(json, Is.EqualTo("{\"name\":\"spud\",\"furColor\":\"brindle\",\"dogTag\":null,\"age\":5}"));
+        }
+
+        [Test]
+        public void DeserializeObject_generic_deserializes_json_using_CompactSerializerSettings()
+        {
+            var dogJson = "{\"name\":\"Barney\",\"furColor\":\"brindle\",\"age\":10}";
```

Comment:
Because it took compact input it deserialized with the CompactSerializerSettings? Is there anything a bit more concrete/unique about the deserialization that can be tested?